tests/private/parse_whl_name/parse_whl_name_tests.bzl

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
```

Comment:
I'd possibly suggest a few tests for `macos`? They're a bit funky as far as I remember?

`numpy` probably has a good example of wheels that exist? https://pypi.org/project/numpy/#files

Reply:
Added a real example, but the spec of wheel naming is pretty settled.